README.md

```diff
@@ -61,7 +61,7 @@ Every call to the openshift installer cli must have at least one `--cluster` opt
 * Mandatory parameters:
   * name: The name of the cluster
   * platform: The platform to deploy the cluster on (supported platforms are: aws, rosa and hypershift)
-  * region: The region to deploy the cluster
+  * region: The region to deploy the cluster. Optional for AWS-IPI and AWS-OSD clusters, but mandatory for other (GCP, ROSA, Hypershift) clusters.
```

Comment:
why it is mandatory for `ROSA, Hypershift`?

Reply:
This was discussed that auto-region is only for AWS clusters. Do we need this for ROSA/Hypershift as well (I may have missed it then)??

Reply:
Sorry, you are right (It was me that said it :-) )